_pages/install/site.adoc

```diff
@@ -0,0 +1,348 @@
+** `--no-install-fonts` - Do not install fonts
+** `--skip-install-fonts` - Skip font installation prompt
+
+* `--continue-without-fonts` - Proceed when fonts are missing
+** `--no-continue-without-fonts` - Stop when fonts are missing
+** `--skip-continue-without-fonts` - Skip missing fonts prompt
+
+* `-s, --stylesheet FILE` - Custom stylesheet for HTML output
+
+* `-t, --template-dir DIR` - Custom Liquid templates for site design
+
+* `-S, --strict` - Abort compilation on errors
+** `--no-strict` - Continue on errors
+** `--skip-strict` - Skip strict mode prompt
+
+== Configuration
+
+The site generation can be customized using a YAML configuration file:
+
+[source,yaml]
+----
+metanorma:
+  source:
+    files:           # Array of files to process
+      - doc1.adoc
+      - "series/*.adoc"
+  collection:
+    name: "Site Title"           # Site title
+    organization: "Publisher"    # Site owner
+  template:
+    path: "templates"            # Custom template directory
+    stylesheet: "custom.css"     # Custom stylesheet
+    output_filename: ""          # Filename template
+----
+
+== Templates and Styling
+
+=== General
+
+Templates are written in link:https://shopify.github.io/liquid/[Liquid^],
+a simple templating language.
+
+Read on for more information on how to customize the site's HTML output
+using Liquid templates.
+
+=== Template Directory Structure
+
+The `-t, --template-dir` option allows customization of the site's HTML output
+using Liquid templates.
+The template directory must contain:
+
+==== Required Files
+
+* `_index.liquid` - Main template for the site's index page
+* `_document.liquid` - Template for individual document entries
+** The name of this file actually depends on how it is referenced in the index
+template.
+See official documentation on
+link:https://shopify.github.io/liquid/tags/template/#include[include^]
+(which is being deprecated)
+and
+link:https://shopify.github.io/liquid/tags/template/#render[render^]
+for more information.
+* CSS stylesheets (referenced in templates)
+
+==== Template Variables
+
+The index template (`_index.liquid`) has access to:
+
+* `title` - Collection title from configuration
+* `author` - Collection author/organization
+* `date` - Generation date
+* `css` - Stylesheet content
+* `metanorma_v` - Metanorma version (if available)
+* `documents` - Array of document entries
+
+The document template (`_document.liquid`) has access to:
+
+* `document.docid.id` - Document identifier
+* `document.title` - Document title
+* `document.html` - HTML file path
+* `document.pdf` - PDF file path
+* `document.doc` - Word document path
+* `document.xml` - XML file path
+* `document.rxl` - Relaton XML file path
+* `document.uri` - Document URI
+* `document.doctype.type` - Document type
+* `document.edition.content` - Edition information
+* `document.docstatus.stage` - Document status
+* `document.revdate` - Revision date
+
+==== Example Templates
```

Comment:
Examples (unless substantive) should always use `[example]` syntax. This one should not be a separate section.